java/pom.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.alibaba.graphar</groupId>
+    <artifactId>gar-java</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>gar-java</name>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <fastffi.revision>0.1.2</fastffi.revision>
+        <native.library.name>gar-java</native.library.name>
+
+    </properties>
+
+    <dependencies>
+        <!-- The FFI annotation -->
+        <dependency>
+            <groupId>com.alibaba.fastffi</groupId>
+            <artifactId>ffi</artifactId>
+            <version>${fastffi.revision}</version>
+        </dependency>
+        <!-- The FFI annotation processor for code generation -->
+        <dependency>
+            <groupId>com.alibaba.fastffi</groupId>
+            <artifactId>annotation-processor</artifactId>
+            <version>${fastffi.revision}</version>
+        </dependency>
+        <!-- The runtime component of LLVM4JNI -->
+        <dependency>
+            <groupId>com.alibaba.fastffi</groupId>
+            <artifactId>llvm4jni-runtime</artifactId>
+            <version>${fastffi.revision}</version>
```

Comment:
add `<classifier>${os.detected.classifier}</classifier>` to install correct jar with os platform.